components/Navbar/index.js

```diff
@@ -32,50 +39,51 @@ export default function Navbar() {
         setFirestoreUser(u)
       })
   }
+  
   useEffect(() => {
-    saveReferralCookie().then(getUser)
+     getUser()
   }, [user])
 
   return (
-    <>
-      <div className="mx-auto h-full max-w-7xl px-6 ">
-        <nav className="hidden bg-gray-50 dark:bg-black-300 xl:block">
-          <div className="flex items-center justify-between">
-            <div>
-              <Link href="/">
-                <div className="flex items-center justify-center text-black-300 transition ease-in hover:text-primary-300 hover:no-underline dark:text-white-100 dark:hover:text-primary-300">
-                  <Image width={42} height={42} src="/assets/img/w3d-logo-symbol-ac.svg" />
 
-                  <p className="hidden cursor-pointer pl-3 text-xl font-bold leading-normal sm:block">
-                    WEB3DEV
-                  </p>
-                </div>
-              </Link>
-            </div>
+    <>  
+        <Navbar variant={'floating'} isBordered={true} >
+            <Link href="/">
+              <Navbar.Brand css={{ gap:'$5' }} >
+                <Image width={42} height={42} src="/assets/img/w3d-logo-symbol-ac.svg" />
+                <Text weight={'bold'}>WEB3DEV</Text>
+                <ThemeSwitch />
+              </Navbar.Brand>
+            </Link>
 
-            {process.env.NEXT_PUBLIC_ENVIRONMENT && (
-              <div className="w-1/3 bg-gradient-to-r from-green-400 to-violet-500 text-center font-semibold tracking-widest">
+            {/* <Navbar.Content hideIn={'md'} >
+              {process.env.NEXT_PUBLIC_ENVIRONMENT && (
```

Comment:
we need to avoie commented code along the code, are we going to use this?

Reply:
This is one development state refear on Navbar, and make sense to maintain this code ! Thanks 😄

Reply:
Can you explain when/why are we using it? just curious 😄